### ISSUE TITLE
The setting of the number of epochs

### ISSUE THREAD

**Issue by JackLiu2025** [cq-dong/DFT_25#2]
I have a question regarding the setting of the number of epochs. Why did you choose 75 epochs in the model? Is this an empirical value? 
Additionally, the MASTER paper also used a similar mechanism to control train loss, and it seems that models using early stopping strategies tend to perform relatively weaker. How should one determine the number of training epochs in practice? 
The performance improvement of the model made me suspect the use of future data at one point, LOL :) Congratulations to the authors for discovering such an effective method.

**Comment by JackLiu2025** [comment]
I know part of the reason is that the validation set is too short, which may not be representative enough.

**Comment by ToughStyle** [comment]
Have you reproduced the results in the paper?

**Comment by ToughStyle** [comment]
> I have a question regarding the setting of the number of epochs. Why did you choose 75 epochs in the model? Is this an empirical value? Additionally, the MASTER paper also used a similar mechanism to control train loss, and it seems that models using early stopping strategies tend to perform relatively weaker. How should one determine the number of training epochs in practice? The performance improvement of the model made me suspect the use of future data at one point, LOL :) Congratulations to the authors for discovering such an effective method.

Have you reproduced the results in the paper?

**Comment by JackLiu2025** [comment]
> > I have a question regarding the setting of the number of epochs. Why did you choose 75 epochs in the model? Is this an empirical value? Additionally, the MASTER paper also used a similar mechanism to control train loss, and it seems that models using early stopping strategies tend to perform relatively weaker. How should one determine the number of training epochs in practice? The performance improvement of the model made me suspect the use of future data at one point, LOL :) Congratulations to the authors for discovering such an effective method.
> 
> Have you reproduced the results in the paper?
Kind of, using the DATA from MASTER

**Comment by cq-dong** [comment]
The main reference for the experimental setup is MATCC. In the experiment, it appears that the training method of MATCC performs better than MASTER

**Comment by cq-dong** [comment]
> I have a question regarding the setting of the number of epochs. Why did you choose 75 epochs in the model? Is this an empirical value?我有一个关于 epoch 数设置的问题。为什么在模型中选择了 75 个 epoch？这是一个经验值吗？ Additionally, the MASTER paper also used a similar mechanism to control train loss, and it seems that models using early stopping strategies tend to perform relatively weaker. How should one determine the number of training epochs in practice?此外，MASTER 论文还使用了类似的机制来控制列车损失，似乎使用早停策略的模型往往表现相对较弱。在实践中应该如何确定训练 epoch 的数量？ The performance improvement of the model made me suspect the use of future data at one point, LOL :) Congratulations to the authors for discovering such an effective method.模型的性能改进让我一度怀疑使用了未来数据，哈哈:)祝贺作者发现了这样一种有效的方法。

The main reference for the experimental setup is MATCC. In the experiment, it appears that the training method of MATCC performs better than MASTER  （see readme）

**Comment by cq-dong** [comment]
> I know part of the reason is that the validation set is too short, which may not be representative enough.我知道部分原因是验证集太短，这可能不够具有代表性。


The validation set in the paper is three months, and the test set is approximately three years
